### DISCUSSION TITLE
Need help configuring - about to lose my mind!!

### DISCUSSION THREAD

**Discussion by glenniana18** [merbanan/rtl_433#2986] [Q&A]
I have spent 3 days trying to configure this add-on and I feel like such an idiot that I can't get it working.  Could someone please help me out??  

My config file is below, hacked together based on what I could understand:

```
output      mqtt://xxx.xxx.xx.xxx:1883,user=xxx,pass=xxx

frequency   433.92M
protocol -R 41 -R 192 -R 231

convert     si
report_meta newmodel
```

My log shows the following:

```
[22:09:34] WARNING: rtl_433 now supports automatic configuration and multiple radios. The rtl_433_conf_file option is deprecated. See the documentation for migration instructions.
Starting rtl_433 -c /config/rtl_433/rtl_433.conf
rtl_433 version 23.11-140-g367378fa branch master at 202406230725 inputs file rtl_tcp RTL-SDR
MQTT: Publishing MQTT data to 192.168.50.129 port 1883
MQTT: Publishing device info to MQTT topic "rtl_433/9b13b3f4-rtl433-next/devices[/type][/model][/subtype][/channel][/id]".
MQTT: Publishing events info to MQTT topic "rtl_433/9b13b3f4-rtl433-next/events".
MQTT: Publishing states info to MQTT topic "rtl_433/9b13b3f4-rtl433-next/states".
Disabling all device decoders.
newmodel option (-M) is deprecated.
Use "-F log" if you want any messages, warnings, and errors in the console.
Found Rafael Micro R820T tuner
Exact sample rate is: 250000.000414 Hz
[R82XX] PLL not locked!
```

I am using Mosquitto Broker (which i've confirmed is set up correctly) and MQTT Explorer, and it isn't seeing my devices :(

Can anyone help me out?  I am happy to provide any other data if needed.

**Comment by glenniana18** [comment]
I have a follow-up question to this, since I'm desperate for solutions.
My USB dongle is a Nooelec RTL-SDR v5.  I have it connected to my QNAP NAS which is also running HA as a virtual machine.  I have the NAS set to pass through the Nooelec to the VM.  Is that enough?  Do I need to install some type of driver for the Nooelec, or will RTL_433 see it as-is?  
Appreciate any thoughts!!

**Comment by gdt** [comment]
This is the repository for the rtl_433 program.  You are, I think, having trouble configuring a "home assistant add-on" which is a different  program (which uses rtl_433).    The addon is at https://github.com/pbkhrv/rtl_433-hass-addons

To figure this out, you need to debug the pieces rather than just saying the whole thing doesn't work.   That involves running rtl_433 with output just to the temrinal, without any -R options, and it involves publishing to your broker with mosquitto_pub (acls are hard).   But how to do that within the addon, which intends to wrap rtl_433 and make it easy -- but wrapping makes debugging harder -- should be asked of the addon project.
